### ISSUE TITLE
fontawesome

### ISSUE THREAD

**Issue by martunta** [dwyl/image-uploads#6]
looks like old package is no longer available, thus project does not work after cloning git repo

**Comment by nelsonic** [comment]
Hi @markizs!
The package still appears to be on Atmosphere: https://atmospherejs.com/fortawesome
Please submit a Pull Request removing fontawesome from smart.json:
https://github.com/ideaq/image-uploads/blob/master/smart.json#L4
(its not really a *requirement* to have icons on buttons...)
Thanks!

**Comment by markizs** [comment]
yeah, I managed to remove fontawesome4, update meteor and add fortawesome:fontawesome instead and then I got further. 

Thought something still did not work with imagemagick, so I could not get repo to work :/ Not sure what I am doing wrong, got this: 

``W20150222-17:03:33.776(2)? (STDERR) Error: Can't find npm module 'Imagemagick'. Did you forget to call 'Npm.depends' in package.js within the 'npm' package?
``

but apt-get install imagemagick shows that it is on latest version. I am using ubuntu 14.10
